tetris.js

```diff
@@ -431,4 +431,15 @@ module.exports = {
 			return arrayPhases[0]
 		}
 	}
-}
\ No newline at end of file
+	class Tetromino {
+		constructor(typeOfTetromino, initialRotationPhase, initialCoordinates) {
+			this.typeOfTetromino = typeOfTetromino
+			this.initialRotationPhase = initialRotationPhase;
+			this.initialCoordinates = initialCoordinates;
+		}
+		tetris.rotateTetromino(typeOfTetromino, rotationPhaseOfTetromino, tetrominoCoordinates)
```

Comment:
parameters typeOfTetromino, rotationPhaseOfTetromino, tetrominoCoordinates are not in the acceptance criteria, please remove them